### ISSUE TITLE
Supabase's auth is experiencing errors.

### ISSUE THREAD

**Issue by erfolgsleiter** [FlutterFlow/flutterflow-issues#1586]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

There was an issue with using supabase's auth, when using Test Mode. My project was working fine 2 weeks ago, but today when I opened the test mode, I encountered an error saying "Expected a value of 'Stream<AuthState>', but got one of type _BehaviorSubjectStream<AuthState>".

Run Mode is possible

Same issue like at first of august 2023 happens again since yesterday (03.10.2023)
https://github.com/FlutterFlow/flutterflow-issues/issues/1197
issue 1197 is already closed.
could not find any recommendations to solve it 

### Expected Behavior

Under normal circumstances, you should be able to access the auth interface smoothly.

### Steps to Reproduce

1.Create a blank project.
2.Simply create a login interface.
3.Open Authentication in the settings and select "supabase" as the Authentication type.
4.Fill in the URL and key in the Supabase interface.
5.Complete the action for the login interface.
6.When everything seems normal and the editor is not showing any errors, the error you described will occur when you open test mode.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

none

### Context

similar like https://github.com/FlutterFlow/flutterflow-issues/issues/1197

### Visual documentation

<img width="168" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/137875393/e71e3ae9-6dfd-4cb4-9de5-bec614cc0738">

### Additional Info

run mode works fine

### Environment

```markdown
- FlutterFlow version: v4.0 - Flutter 3.10.4
- Platform:Web
- Browser name and version: Brave Version 1.58.135 Chromium: 117.0.5938.140
- Operating system and version affected: web windows 10
```

**Comment by hariprasadms** [comment]
Hi @erfolgsleiter - Thanks for submitting the issue. I just tested the above steps in a new project. Supabase create account and login, both the features are working fine.

Re produce the issue in a blank project and send us the project bug report code. We can investigate it for you. Thanks.

**Comment by erfolgsleiter** [comment]
Hello, @hariprasadms 
I did following steps:
- setup a new project with supabase authentication of the same supabase project--> worked fine without problems.
- duplicated the project with the error 
- disable and enable supabase authentication in auth settings
- deleted and renewed supabase project API URL and anon key
-  removed one page at a time run test mode after removing each page until only the "onboarding" page was left. the error is still present.

here is the bug report code I copied from the scaffold "onboarding" page:

IT4wlcmAz5VNtbxa15D+dMdFmikWCDs1U4JIjupEFSs8GZT0B4kcYs+kfklUOfW7S1ZLeVqZgXwX/MrTvfztA/E3Cyutfo8+z85qTQvgbGKQMrq3Or63b3B9N8NQIFy44cOwkwp5Nfp0V3wt7zn3AOm/TgnqN4DLDEgvAOaoXdeK2SrDX1iXc2URm05KZDPz

I hope the error can be solved. spend a lot of time in this project
Thank you very much
